javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,39 @@
+const body = document.querySelector('body');
+const cityName = document.getElementById('name') ;
+const button = document.getElementById('btn');
+const p = document.getElementById("cityname")
+const weatherOfCity = () => { 
```

Comment:
Consider breaking down this function to a few small functions and using empty lines and spaces to make it more readable.